### DISCUSSION TITLE
Confused with ocr training about  Text Detection and Text Recognition

### DISCUSSION THREAD

**Discussion by chocolatedesue** [PaddlePaddle/PaddleOCR#7818] [Q&A]
I am new to ocr and happy to find this awesome works ! 

Acrodding to [tutorials](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/recognition_en.md), the training and infering of the  Text Recognition don't need the detection model while in the deployment both model are needed. 

Since I can get the texts directly from the Text Recognition, why the detection model is needed ?

What is the relationship between two of them ?

Hope for your reply !

**Comment by chocolatedesue** [comment]
I figure it out with aistudio courses. 

detection model  is used to quickly get text from images

**Reply by sumitclearquote** [reply]
@chocolatedesue Hi, How does the recognition model train without detecting the text first? Could you elaborate on how you trained the text detection and text recognition models.
